sample/.github/workflows/android_deploy_production.yml

```
@@ -34,9 +35,9 @@ jobs:
 
       - name: Set up .env
         env:
-          ENV_PRODUCTION: ${{ secrets.ENV_PRODUCTION }}
+          ENV: ${{ secrets.ENV }}
         run: |
-          echo $ENV_PRODUCTION > .env
+          echo "$ENV" > .env
```

I think we can apply the same approach as the above scripts
```suggestion
          echo -e ENV >> .env
```